### DISCUSSION TITLE
LXQT heavier and menu-bloated

### DISCUSSION THREAD

**Discussion by PlainUser** [lxqt/lxqt#2200] [Ideas]
![LXQT](https://user-images.githubusercontent.com/100782667/157514125-ffc48ade-159f-4ca7-bccb-2078af987fe7.png)
![LXDE](https://user-images.githubusercontent.com/100782667/157519942-20f24423-63b1-49e9-9d69-b8104725fce8.png)
Hi, 
This is my humble opinion, I am not a developer, I am not a power user, just a plain linux user. In my search for fast, simple and light linux OS, I found LXDE. I understand LXDE now is LXQT, sadly I have to say that LXQT is heavier and menu-bloated compared to LXDE. I installed LXDE-core in debian (net with no apps installed, no browser installed) and the RAM usage is 171mb, while in debian with LXQT the RAM usage is 371. LXDE is still lighter, simplier, friendlier. LXqT has so many configurations, i got lost. So many menus( a menu, submenu). I will keep using LXDE. I have to say LXDE is great, modular, light, simple and fast.

@GreenLunar , I have to recognize now that LXDE-core uses more RAM than LXQT-core. Both with only gdebi, synaptic, chromium and onlyoffice. Thanks for the comments and replies.
Though I personally still like a more simple menu, but that is a personal preference.

I think the sucess of open source is based on its friendliness and simplicity towards the regular common user.

**Comment by stefonarch** [comment]
> I understand LXDE now is LXQT,

No, some history [here](https://github.com/lxqt/lxqt/wiki/History). 
No one has to use LXQt, as no one _has_ to use any of the existing and supported DE's.
And nowadays for the memory footprint you should rather look at the browser. 

> LXqT has so many configurations, i got lost. So many menus( a menu, submenu)

Funny, usually this judgement is heard regarding Plasma/KDE ;) Surprisingly we are often rejecting options asked for from "regular users"
This said, "the regular common user" is not well defined, everyone has it's habits and tastes.

**Comment by tsujan** [comment]
>  LXQT heavier and menu-bloated

LXDE would become "heavier" too if it was actively developed. What was heavy 10 years ago is considered light now. LXQt uses as little RAM and CPU time as possible but RAM and CPU time are there to be used reasonably.

As for "bloated" menus, LXQt doesn't deprive users of practical options, while it doesn't add impractical options either. If you want GUIs with as little options as possible, Gnome will be your best choice.

**Reply by shlyakpavel** [reply]
Gnome forever! XD

**Comment by GreenLunar** [comment]
@PlainUser may you please check again?

Also worthy to note, LXDE and Razor-qt have made a precedent move and decided to merge the projects into one, LXQt.

[![image](https://user-images.githubusercontent.com/2053884/157458218-49869748-1fcf-4f59-b8fa-387e11826e96.png)](https://upload.wikimedia.org/wikipedia/commons/6/6e/X_window_system_desktop_environments_timeline.svg)

May you share some info and screenshots when and where you see indications for lesser lightweight experience?

Even [Wireshark](https://blog.wireshark.org/2013/10/switching-to-qt/) has moved from GTK+ to Qt and is much more responsive ever since.

**Comment by GreenLunar** [comment]
I would like to add, that I've an interest in LXQt, not because I'm using it (actually, I'm using Xfce for my own reasons), it's because of it's politics.

There are so many distros and so many Linux communities that got things completely wrong and have sometimes lost track, and sometimes, thank god, did more good than bad.

Continue reading at LinuxQuestions.org [On Linux and Religious communities](https://www.linuxquestions.org/questions/general-10/on-linux-and-religious-communities-4175709172/)

**Comment by GreenLunar** [comment]
@PlainUser
Please share with us screenshots which related to UI.
What LXDE has that LXQt is missing?

_P.S. I'm not a developer nor-member of the project.  I'm an LXQt user._

**Comment by GreenLunar** [comment]
On Wed, 09 Mar 2022 11:43:51 -0800
PlainUser ***@***.***> wrote:

> @GreenLunar , I have to recognize now that LXDE-core uses more RAM
> than LXQT-core. Both with only gdebi, synaptic, chromium and
> onlyoffice. Thanks for the comments and replies. Though I personally
> still like a more simple menu, but that is a personal preference.

This is great.

You're not the only one; the philosophy of the SXMO project is just
like yours, both in regard to menu and preferences.

Concerning to preferences or menus, you might want to try qml-launcher
or other application launcher as a solution.

https://github.com/alamminsalo/qml-launcher
